src/preact/base-element.js

```diff
@@ -398,6 +406,7 @@ export class PreactBaseElement extends AMP.BaseElement {
    */
   checkMutations_(records) {
     const Ctor = this.constructor;
+    this.mutationObserverCallback(records);
```

Comment:
let's either remove `records` from here,  or add them in the `@param` of the `mutationObserverCallback`.

Reply:
Good point. Removed for now - we can add them back in when we find a reason to.